packages/cli/src/models/Verifier.js

```diff
@@ -47,4 +52,103 @@ async function publishToEtherchain(params) {
   }
 }
 
+async function publishToEtherscan(params) {
+  if (params.apiKey === undefined) {
+    throw new Error('API key not specified.')
+  }
+
+  const { network, compilerVersion, optimizer, contractAddress } = params
+  const compiler = `v${compilerVersion.replace('.Emscripten.clang', '')}`
+  const optimizerStatus = optimizer ? 1 : 0
+
+  let apiSubdomain = ''
+
+  if (network === 'mainnet') {
+    apiSubdomain = 'api'
+  } else if (network === 'rinkeby') {
+    apiSubdomain = 'api-rinkeby'
+  } else if (network === 'ropsten') {
+    apiSubdomain = 'api-ropsten'
+  } else if (network === 'kovan') {
+    apiSubdomain = 'api-kovan'
+  } else {
+    throw new Error('Invalid network. Currently, etherscan supports mainnet, rinkeby, ropsten and kovan')
+  }
+
+  const etherscanApiUrl = `https://${apiSubdomain}.etherscan.io/api`
+  const etherscanContractUrl = `https://${network}.etherscan.io/address`
+
+  const checkVerificationStatus = async (guid) => {
+    const queryParams = querystring.stringify({
+      guid,
+      action: 'checkverifystatus',
+      module: 'contract',
+    })
+
+    try {
+      const response = await axios.request({
+        method: 'GET',
+        url: `${etherscanApiUrl}?${queryParams}`,
+      })
+
+      if (response.status === 200) {
+        if (response.data.status === '1') {
+          return true
+        } else {
+          return false
+        }
+      } else {
+        throw new Error(`Error while trying to verify contract; ${response.data.status}; ${response.data.message}; ${response.data.result}`)
```

Comment:
Showing all the etherscan's messages could be a bit annoying. 
```console
foo@bar:~$ Error while trying to verify contract; 0; NOTOK; Missing or invalid ApiKey
```
Let's remove the `response.data.status` and the `response.data.message`, and only keep the `response.data.result`.